packages/usa-accordion/src/index.js

```diff
@@ -82,17 +82,26 @@ const accordion = behavior(
   },
   {
     init(root) {
-      select(BUTTON, root).forEach((button) => {
-        const expanded = button.getAttribute(EXPANDED) === "true";
-        toggleButton(button, expanded);
-      });
+      // Check if Banner has previously initialized accordion.
+      if (!this.hasInit) {
```

Comment:
Is this something which could make sense to implement as common in the `behavior` function initialization for all components?

Reply:
I'm conflicted on this and would prefer to limit this feature for now. Mainly to test and be able to have the ability to quickly change if needed.

That said, I'm also looking forward to the teams feedback. If it's something we should move forward with on all components, that's totally possible too.

Reply:
I'm comfortable with leaving this scoped to these specific components for now. But I agree it could make sense to move into `behavior` as we identify the need.